### ISSUE TITLE
Update Documentation for DEV_DSF_BPE_PROCESS_EXCLUDED

### ISSUE THREAD

**Issue by UKFr-DIZ** [datasharingframework/datasharingframework.github.io#17]
Hi,

for the [documentation ](https://github.com/datasharingframework/datasharingframework.github.io/blob/main/docs/src/v1.1.0/maintain/bpe/configuration.md#dev_dsf_bpe_process_excluded) of the DSF BPE and the Configuration DEV_DSF_BPE_PROCESS_EXCLUDED, it'd be nice if you add an example and mention that you can excluded specific version of process. 

The new format "process-id|version" [code](https://github.com/datasharingframework/dsf/blob/195d03f73880f5edf9be6b6e936dbf3861c3d042/dsf-bpe/dsf-bpe-server/src/main/java/dev/dsf/bpe/plugin/ProcessIdAndVersion.java#L20) should be shown in the example too. 
It used to be different, where `/` was used to split the process ID and the process version. 

Suggestion: 

```markdown
### DEV_DSF_BPE_PROCESS_EXCLUDED
- **Property:** dev.dsf.bpe.process.excluded
- **Required:** No
- **Description:** List of process names that should be excluded from deployment during startup of the DSF BPE server; comma or space separated list, YAML block scalars supported. Each process can be version specifically splitted in the format ´id|version`
- **Recommendation:** Only deploy processes that can be started depending on your organization's roles in the Allow-List
- **Example:** `dsfdev_updateAllowList|1.0, another_process, other_process|0.5`
```

**Comment by schwzr** [comment]
will be fixed in the upcoming release #19

**Comment by schwzr** [comment]
Fixed
